core/domain/question_domain.py

```diff
@@ -79,32 +86,31 @@ def __init__(self, change_dict):
                 self.old_value = change_dict['old_value']
             else:
                 raise Exception('Invalid change_dict: %s' % change_dict)
-
-        if self.cmd == CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION:
+        elif self.cmd == CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION:
             self.question_dict = change_dict['question_dict']
             # Note that change_dict['skill_id'] may be None if this change is
             # being done in the context of a suggestion.
             self.skill_id = change_dict['skill_id']
+        elif self.cmd == CMD_MIGRATE_STATE_SCHEMA_TO_LATEST_VERSION:
```

Comment:
Add these also to the docstring for this function.

Reply:
fixed